README.md

```diff
@@ -1,30 +1,27 @@
 [![CI tests](https://github.com/emmo-repo/domain-crystallography/workflows/CI%20tests/badge.svg)](https://github.com/emmo-repo/domain-crystallography/actions/)
 
+# Battery INterFace Ontology (BattINFO)
```

Comment:
I have noticed that it is common to use the hash in headers instead of underline. What is the reason for that?

Personally I feel that the underline is more readable without rendering.

Reply:
The reason is to make it easier to determine 1st, 2nd, 3rd, 4th... header (number of preceding hashes).
Another reason might be that it saves space and bytes. While it might be more difficult to read in some editors, the rendered result is the same.
Since it was already used in some markdown files (and in some it was even mixed between this way of making headers and the underline way) I simply streamlined it, i.e., used a single way of doing it - and then chose my personal favorite way.

If you think it makes more sense with the underlining it could be changed to that throughout.

Reply:
We go for consistency